includes/Modules/Analytics_4/Web_Tag.php

```diff
@@ -24,6 +25,18 @@ class Web_Tag extends Analytics_Web_Tag {
 
 	use Method_Proxy_Trait;
 
+	/**
+	 * Constructor.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $tag_id Tag ID.
+	 * @param string $module_slug Module slug.
+	 */
+	public function __construct( $tag_id, $module_slug ) {
+		Module_Tag::__construct( 'G-' . $tag_id, $module_slug );
```

Comment:
@Hazlitte in PHP if we need to call/override a parent method we should use the `parent` keyword instead of calling a specific class.

```suggestion
		parent::__construct( 'G-' . $tag_id, $module_slug );
```

Reply:
👍